cmd/groundstation/main.go

```diff
@@ -10,29 +10,28 @@ import (
 )
 
 func main() {
-	slog.Info("Starting groundstation")
-
 	configuration, err := config.GetConfiguration("config.toml")
 
 	if err != nil {
 		slog.Error("failed to load config, shutting down", "err", err)
 		return
 	}
 
-	srv := groundstation.NewServer()
-
 	// open database connection
 	dbUrl := configuration.Database.Url
-	_, err = postgres.New(dbUrl)
+	db, err := postgres.New(dbUrl)
 	if err != nil {
 		slog.Error("when opening database", "err", err)
 		return
 	} else {
 		slog.Info("connected to database", "url", dbUrl)
 	}
 
-	slog.Info("Starting groundstation API server", "port", configuration.Server.Port)
+	// start servers
+	slog.Info("Starting groundstation dish server")
```

Comment:
NIT: wait, where did this terminology come from

Reply:
The API server had a name (api) and a message indicating it had booted, so I thought the satellite->groundstation could use one too.
Happy to take suggestions on what we call the communication from satellite->groundstation, but I personally do quite like dish (it's like the reciever/transmitter to satellites on a groundstation)

Reply:
Alternatively, we could make the "groundstation api server", just "api server" and make the satellite->groundstation server the "groundstation server"

Reply:
Can we bikeshed this another day please!!

Reply:
Of course. I'm sure we'll have a wheely great time